bindings/nodejs/lib/types/block/address.ts

```diff
@@ -118,12 +133,107 @@ class NftAddress extends Address {
     }
 }
 
+/**
+ * An implicit account creation address.
+ */
+class ImplicitAccountCreationAddress extends Address {
+    /**
+     * The Ed25519 address.
+     */
+    @Transform(({ value }) => new Ed25519Address(value), { toClassOnly: true })
+    @Transform(({ value }) => value.toString(), { toPlainOnly: true })
+    private pubKeyHash: Ed25519Address;
```

Comment:
Tbh at this point I think I agree with Brord, I'd have a HexEncodedString field and a Ed25519Address field.
This is just too convoluted for nothing.